<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demonstration_Window</class>
 <widget class="QMainWindow" name="Demonstration_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demonstration_Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="pic_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionstart"/>
    <addaction name="actionstop"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>路径选择</string>
    </property>
    <addaction name="actionpath1"/>
    <addaction name="actionpath2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionstart">
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionpath1">
   <property name="text">
    <string>path1</string>
   </property>
  </action>
  <action name="actionpath2">
   <property name="text">
    <string>path2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
